### PR TITLE
Add Seville children stations

### PR DESCRIPTION
**Pull request by chtitux**
This pull request has been automatically generated by Station Studio

### CODE REVIEW

File: stations.csv [hunk 1]
```diff
@@ -5269,7 +5269,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;is_city;country;
 6680;Las Rosas;las-rosas;;;;;;t;ES;f;Europe/Madrid;f;ESRSS;;f;;f;;f;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
 6681;San Fernando de Cádiz;san-fernando-de-cadiz;7151402;;36.4758;-6.19444;;f;ES;f;Europe/Madrid;t;ESSFD;;f;;f;;f;;f;;f;;f;;f;;f;51402;t;f;;;;;;;;;;;;;;;;;;
 6682;Salamanca;salamanca;7130100;;40.971547;-5.648659;;f;ES;f;Europe/Madrid;t;ESSLM;;f;;f;7100027;f;;f;;f;;f;;f;;f;30100;t;f;;;;;;;;;;;;;;;;;;
-6683;Sevilla;sevilla;;;;;;t;ES;f;Europe/Madrid;f;ESSVQ;;f;;f;;f;;f;;f;;f;;f;;f;;f;f;;;Seville;;Séville;Siviglia;;;;;;;Sewilla;Sevilha;Севилья;;;
+6683;Sevilla;sevilla;;;37.389254195966004;-5.984458923339844;;t;ES;f;Europe/Madrid;t;ESSVQ;;f;;f;;f;;f;;f;;f;;f;;f;;f;f;;;Seville;;Séville;Siviglia;;;;;;;Sewilla;Sevilha;Севилья;;;
```
Comment (mckbrmn) [inline stations.csv]: Much latitude. Very longitude. Such precise.

File: stations.csv [hunk 2]
```diff
@@ -22860,8 +22860,8 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;is_city;country;
...
 24417;Serin;serin;7115302;;43.504815;-5.782733;;f;ES;f;Europe/Madrid;t;;;f;;f;7115302;f;;f;;f;;f;;f;;f;15302;t;f;;;;;;;;;;;;;;;;;;
 24418;Setenil;setenil;7155005;;36.863306;-5.12155;;f;ES;f;Europe/Madrid;t;;;f;;f;7155005;f;;f;;f;;f;;f;;f;55005;t;f;;;;;;;;;;;;;;;;;;
-24419;Sevilla Santa Justa;sevilla-santa-justa;7151003;;37.391925;-5.975264;;f;ES;f;Europe/Madrid;t;ESAGN;;f;;f;7100180;f;;f;;f;;f;;f;;f;51003;t;f;;;Seville;;Séville;Siviglia;;;;;;;Sewilla;Sevilha;Севилья;;;
-24420;Sevilla-Virgen del Roció;sevilla-virgen-del-rocio;7151110;;37.362827;-5.975831;;f;ES;f;Europe/Madrid;t;;;f;;f;7151110;f;;f;;f;;f;;f;;f;51110;t;f;;;Seville;;Séville;Siviglia;;;;;;;Sewilla;Sevilha;Севилья;;;
+24419;Sevilla Santa Justa;sevilla-santa-justa;7151003;;37.391925;-5.975264;6683;f;ES;f;Europe/Madrid;t;ESAGN;;f;;f;7100180;f;;f;;f;;f;;f;;f;51003;t;f;;;Seville;;Séville;Siviglia;;;;;;;Sewilla;Sevilha;Севилья;;;
+24420;Sevilla-Virgen del Roció;sevilla-virgen-del-rocio;7151110;6683;37.362827;-5.975831;;f;ES;f;Europe/Madrid;t;;;f;;f;7151110;f;;f;;f;;f;;f;;f;51110;t;f;;;Seville;;Séville;Siviglia;;;;;;;Sewilla;Sevilha;Севилья;;;
```
Comment (mckbrmn) [inline stations.csv]: Wrong column here!

**Comment by chtitux** [comment]
Indeed, long/lat truncated and column fixed!

File: stations.csv [hunk 3]
```diff
@@ -22860,8 +22860,8 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;is_city;country;
 24416;Selgua;selgua;7178300;;41.9061;0.1484;;f;ES;f;Europe/Madrid;t;;;f;;f;;f;;f;;f;;f;;f;;f;78300;t;f;;;;;;;;;;;;;;;;;;
 24417;Serin;serin;7115302;;43.504815;-5.782733;;f;ES;f;Europe/Madrid;t;;;f;;f;7115302;f;;f;;f;;f;;f;;f;15302;t;f;;;;;;;;;;;;;;;;;;
 24418;Setenil;setenil;7155005;;36.863306;-5.12155;;f;ES;f;Europe/Madrid;t;;;f;;f;7155005;f;;f;;f;;f;;f;;f;55005;t;f;;;;;;;;;;;;;;;;;;
-24419;Sevilla Santa Justa;sevilla-santa-justa;7151003;;37.391925;-5.975264;;f;ES;f;Europe/Madrid;t;ESAGN;;f;;f;7100180;f;;f;;f;;f;;f;;f;51003;t;f;;;Seville;;Séville;Siviglia;;;;;;;Sewilla;Sevilha;Севилья;;;
-24420;Sevilla-Virgen del Roció;sevilla-virgen-del-rocio;7151110;;37.362827;-5.975831;;f;ES;f;Europe/Madrid;t;;;f;;f;7151110;f;;f;;f;;f;;f;;f;51110;t;f;;;Seville;;Séville;Siviglia;;;;;;;Sewilla;Sevilha;Севилья;;;
+24419;Sevilla Santa Justa;sevilla-santa-justa;7151003;;37.391925;-5.975264;6683;f;ES;f;Europe/Madrid;t;ESAGN;;f;;f;7100180;f;;f;;f;;f;;f;;f;51003;t;f;;;Seville;;Séville;Siviglia;;;;;;;Sewilla;Sevilha;Севилья;;;
```
Comment (paulRbr) [inline stations.csv]: Please mark this stations as `is_main_station`

Comment (chtitux) [inline stations.csv]: Sevilla Santa Justa is now marked as main station.

Commit message has been rewritten too.

**Comment by paulRbr** [comment]
LGTM

**Pull request merged**